### PR TITLE
Fix dashboard layout

### PR DESCRIPTION
**Pull request by SidharthK2**
# Fix navbar issues in mobile breakpoint

Moved IQ button to sidebar on mobile, removed down chevron from language selector. Also fixed IQ button spamming API.

Before:
image

After:
image
image

closes https://github.com/EveripediaNetwork/issues/issues/1804

### CODE REVIEW

File: src/components/dashboard/IQButton.tsx [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+} from '@chakra-ui/react'
 import React, { useEffect, useState } from 'react'
 import { addToken } from '@/utils/add-new-token'
 import Image from 'next/image'
 import * as Humanize from 'humanize-plus'
 import axios from 'axios'
 
-export const IQButton = () => {
+export const IQButton = (props: ButtonProps) => {
+  const sigFigs = useBreakpointValue({ base: 4, md: 4 })
```
Comment (Aliiiu) [inline src/components/dashboard/IQButton.tsx:16]: The value does not change still, why not use 

`const sigFigs = useBreakpointValue({ base: 4 })`

Comment (Aliiiu) [inline src/components/dashboard/IQButton.tsx:16]: why not use 

`const sigFigs = useBreakpointValue({ base: 4})`

**Review by Aliiiu — CHANGES_REQUESTED** [review]

File: src/components/dashboard/IQButton.tsx (hunk 1 above)
Comment (SidharthK2) [reply src/components/dashboard/IQButton.tsx:16]: I am not sure if we want to use different rounding on smaller breakpoints. What do you think?

Comment (Aliiiu) [reply src/components/dashboard/IQButton.tsx]: Alright. I think the code is redundant since the sigFigs value is not changing for the two breakpoint

**Review by Aliiiu — APPROVED** [review]

**Review by Simplypheyie96 — APPROVED** [review]

**Pull request merged**